### PR TITLE
New backend & frontend setup

### PR DESCRIPTION
**Pull request by vagechirkov**
This is still a work in progress, but I think we can push it to the main branch and finish the ToDo list with a new PR.

@LBrinkmann let me know if this setup looks good to you and I can merge it into the main branch.

Done:
- [x] Most of React components
- [x] StoryBook for most of React component
- [x] Most of the Backend setup
- [x] Setup of most frontend-backend interactions
- [x] Deployment setup
- [x] Add social learning trials (backend and frontend)

ToDo:
- [ ] Add persisting state between trial page reloads
- [ ] Save data after each trial

### CODE REVIEW

**Review by LBrinkmann — COMMENTED** [review]
Looks great. All comments are not urgent, in particular as you are currently working on getting the actual flow of the experiment together. Just something to think about before we move to run the actual pilots.

File: backend/app/routes/session.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        subject = subjects_with_id[0]
+        # get session for the subject
+        session = await Session.find_one(Session.subject_id == subject.id)
+
+    trial = session.trials[session.current_trial_num]
+    return session, trial
+
+
+async def initialize_session(subject: Subject) -> Session:
+    # get any available session
```
[the remark this answers is not in the document]
Comment (vagechirkov) [reply backend/app/routes/session.py:139]: Great point! I was able to reproduce this problem using `pytest`. I will update the session initialization script in the next PR.

[the remark this answers is not in the document]
Comment (vagechirkov) [reply backend/app/routes/session.py:139]: Solved in this commit 2c3aab826cbb85d03181cd1758d21b28479c002c

File: backend/app/routes/session.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        # save session
+        await session.save()
+
+    return {
+        "message": "Trial saved"
+    }
+
+
+async def update_available_status_child_sessions(session):
+    """ Check if child sessions are available"""
```
[the remark this answers is not in the document]
Comment (vagechirkov) [reply backend/app/routes/session.py:95]: Done in e42d1f2f07db684ce684d5df2b3c596616c749cb

**Pull request merged**